src/js/angular/repositories/ontop-repo.directive.js

```diff
@@ -419,7 +445,7 @@ function ontopRepoDirective($uibModal, RepositoriesRestService, toastr, Upload,
         };
 
         const validateHostName = () => {
-            if ($scope.selectedDriver.driverType !== $scope.genericDriverType) {
+            if ($scope.selectedDriver.driverType !== JdbcDriverType.GENERIC) {
```

Comment:
Can you create a method like `isGenericDriver(type)` and reuse it as this check is performed too often here.
The method can be in the OntopDriverData class.

Reply:
Then you can also expose a boolean property isGenericDriver which to be used in the template where this check is also performed many times.

Reply:
Done